### ISSUE TITLE
Miaomiao – no readings shown when directly connected to watch

### ISSUE THREAD

**Issue by zackallan** [NightscoutFoundation/xDrip#542]
When the Miaomiao reader is directly connected to the smartwatch xDrip+ does not display the incoming values. The bluetooth collector status page shows that the watch receives data every 5 minutes but the readings are not shown neither on the watchface nor on the xDrip+ main screen.
(tested with the latest nightlies)

Watch: Ticwatch E
OS watch: Wear OS: 1.4 Android 8.0.0
Phone: Nexus 6P
OS phone: Android 8.1.0

**Comment by tzachi-dar** [comment]
Where is the collector running?
Did that work before?

**Comment by zackallan** [comment]
The problem occurs when the watch is running the collector. I followed the instructions of the xDrip+ watch guide.
It was the first time that I tried to set up the connection between Miaomiao and the smartwatch. So it has not worked for me yet.
When the engineering mode is enabled I can see the received and sent data of the watch in the bluetooth collector status page. While "force wear collection service" is activated the watch receives data every five minutes. It looks similar to the data which is received by the phone when wear collection is disabled.
